layouts/partials/download/getting-started-linux.html

```diff
@@ -27,7 +27,7 @@ <h4>Setting up O3DE from GitHub</h4>
         <a href="https://o3de.org/docs/welcome-guide/setup/installing-linux/">
           <div class="card mb-3">
             <h4>Install O3DE for Linux</h4>
```

Comment:
While you're at it, could you also add (Preview) in the title?
```
<h4>Install O3DE for Linux (Preview)</h4>
```

Reply:
Looks like Doug already did that in #1246.